lib/common.js

```diff
@@ -68,18 +69,9 @@ exports.hasProperties = function(obj) {
 
 exports.readJSON = function(file) {
   return asp(fs.readFile)(file)
-  .then(function(pjson) {
-    pjson = pjson.toString();
-    // remove any byte order mark
-    if (pjson.startsWith('\uFEFF')) // TODO: https://www.npmjs.com/package/strip-bom
-      pjson = pjson.substr(1);
```

Comment:
Just thought, if there is module to do this - why don't use it? It have tests and [works with `Buffer`](https://github.com/sindresorhus/strip-bom/blob/master/index.js#L9-L12) which should be faster a little.